<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battlefieldForm</class>
 <widget class="QWidget" name="battlefieldForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="gameType_label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>My Fairy:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="myFairyName_label">
             <property name="text">
              <string>Fairy Name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Skill:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="myskill_label">
             <property name="text">
              <string>Skill Name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Blood:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="myblood_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="myCase_label">
           <property name="text">
            <string>case</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="myFig_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Opponent Fairy:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="opponentName_label">
             <property name="text">
              <string>Fairy Name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Skill:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="opponent_skill">
             <property name="text">
              <string>Skill Name</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>Blood</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="opponentBlood_label">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="opponentCase_label">
           <property name="text">
            <string>case</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="opponentFig_label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="start_pushButton">
       <property name="text">
        <string>Start to battle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
